Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "wrangler"
-version = "1.13.0"
-authors = ["The Wrangler Team <wrangler@cloudflare.com>", "Avery Harnish <averyharnish@gmail.com>", "Ashley Lewis <ashleymichal@gmail.com>", "Ashley Williams <ashley666ashley@gmail.com>", "Nat Davidson <davidson@cloudflare.com>"]
+version = "1.14.0"
+authors = ["The Wrangler Team <wrangler@cloudflare.com>", "Avery Harnish <averyharnish@gmail.com>", "Ashley Lewis <ashleymichal@gmail.com>", "Ashley Williams <ashley666ashley@gmail.com>", "Nat Davidson <davidson@cloudflare.com>", "Steve Manuel <nilslice@gmail.com>"]
```

Comment:
i'm not sure we add individual contributors to the author list?

Reply:
yeah if anything this should just be wrangler@cloudflare.com at this point